package.json

```diff
@@ -163,7 +163,8 @@
     "axios": ">=1.7.4",
     "postcss": ">=8.4.33",
     "braces": ">=3.0.3",
-    "micromatch": "^4.0.8"
+    "micromatch": "^4.0.8",
+    "cookie": ">=0.7.0"
```

Comment:
If a later version of Cookie is needed (I assume another library is pulling in an older version), consider using a dev resolution instead

Reply:
there are slightly newer ones but they break

Reply:
changed